experiments/tf_trainer/tf_hub_tfjs/notebook/BiasEvaluation.ipynb

```diff
@@ -0,0 +1,612 @@
+      ],
+      "execution_count": 0,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "The autoreload extension is already loaded. To reload it, use:\n",
+            "  %reload_ext autoreload\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "metadata": {
+        "id": "12LU1AjWr-da",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "auth.authenticate_user()"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "I4dzfYrvr3XT",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "# Download and unzip files used in this colab\n",
+        "!curl -O -J -L https://storage.googleapis.com/civil_comments/fat_star_tutorial/fat-star.zip\n",
+        "!unzip -o fat-star.zip"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "FFFXbLiRrvtz",
+        "colab_type": "code",
+        "outputId": "69de8876-a0d8-4e31-816c-a3c135854faa",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 125
+        }
+      },
+      "cell_type": "code",
+      "source": [
+        "!mkdir -p tfjs_model\n",
+        "!gsutil -m cp -R gs://conversationai-public/public_models/tfjs/v1/* tfjs_model"
+      ],
+      "execution_count": 0,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "Copying gs://conversationai-public/public_models/tfjs/v1/saved_model.pb...\n",
+            "Copying gs://conversationai-public/public_models/tfjs/v1/variables/variables.data-00000-of-00001...\n",
+            "/ [0 files][    0.0 B/  3.9 MiB]                                                \rCopying gs://conversationai-public/public_models/tfjs/v1/variables/variables.index...\n",
+            "Copying gs://conversationai-public/public_models/tfjs/v1/assets/universal_encoder_8k_spm.model...\n",
+            "- [4/4 files][ 32.3 MiB/ 32.3 MiB] 100% Done                                    \n",
+            "Operation completed over 4 objects/32.3 MiB.                                     \n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "metadata": {
+        "id": "0bmiyJR60gDP",
+        "colab_type": "code",
+        "outputId": "ded1805f-f50c-4846-cafb-dfb51d79fa4d",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 35
+        }
+      },
+      "cell_type": "code",
+      "source": [
+        "test_df = pd.read_csv(\n",
+        "    'https://raw.githubusercontent.com/conversationai/unintended-ml-bias-analysis/master/unintended_ml_bias/new_madlibber/output_data/English/intersectional_madlibs.csv')\n",
+        "print('test data has %d rows' % len(test_df_float))\n"
```

Comment:
This line gives me an error, I think you meant `len(test_df)`?

Reply:
Done